### DISCUSSION TITLE
compression

### DISCUSSION THREAD

**Discussion by dev72** [dswd/vpncloud#223] [Q&A]
Hello,
does vpncloud implements some sort of compression?

**Comment by dswd** [comment]
No it does not. I think compression in VPNs is an anti-feature:

1. Even the fastest compression is slow compared to network speed
2. Most of the data you transmit is either encrypted (HTTPS or any SSL/TLS protocol) or compressed (Zip files, Media files, etc) so in 99% compression will not reduce data size
